shell/platform/android/io/flutter/plugin/localization/LocalizationPlugin.java

```diff
@@ -19,11 +22,56 @@
   @NonNull private final LocalizationChannel localizationChannel;
   @NonNull private final Context context;
 
+  @VisibleForTesting
+  final LocalizationChannel.LocalizationMessageHandler localizationMessageHandler =
+      new LocalizationChannel.LocalizationMessageHandler() {
+        @Override
+        public String getStringResource(@NonNull String key, @Nullable String localeString) {
+          Context localContext = context;
+          String stringToReturn = null;
+          Locale savedLocale = null;
+
+          if (localeString != null) {
+            Locale locale = localeFromString(localeString);
+
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+              Configuration conf = new Configuration(context.getResources().getConfiguration());
```

Comment:
nit, spell out `config` or `configuration`.

Reply:
Done. I went with `config` since because this file has already used `config`.